src/coreclr/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets

```diff
@@ -16,8 +16,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DefaultLanguageSourceExtension>.il</DefaultLanguageSourceExtension>
     <Language>IL</Language>
     <TargetRuntime>Managed</TargetRuntime>
+    <DefaultProjectTypeGuid>{F571AB99-FB84-49F4-A0DF-F27AA55F3F3F}</DefaultProjectTypeGuid>
```

Comment:
Do we want to check if it is already set? Like C# and VB do?

Reply:
Yeah I wasn't sure about that. Will do for consistency.